crates/re_types/definitions/rerun/components/image_from_camera.fbs

```diff
@@ -0,0 +1,15 @@
+include "rust/attributes.fbs";
+
+include "../datatypes/mat3x3.fbs";
+
+namespace rerun.components;
+
+// ---
+
+/// Camera projection, from image coordinates to view coordinates.
+struct ImageFromCamera (
```

Comment:
This name doesn't sit quite right with me - it seems reversed with the Archetype.

Following from how this is handled with Transforms:
 - Transform3D (A category of relationship) -> TranslationRotationScale (a concrete relationship) -> {Vec3, Quat, Vec3} (A concrete data representation)

I would expect this to look more like:
- ImageFromCamera (A category of relationship) > Pinhole (A concrete relationship) > Mat3x3 (A concrete data representation)

Reply:
I see -- yeah, `image_from_cam` makes sense as a *field* of Pinhole, because then Pinhole gives it context.

But viewing this component in isolation of the Pinhole context, the struct name is overly generic. There's nothing here that tells us that the 3x3 matrix it contains is a pinhole camera projection model.